lm75bd/lm75bd.c

```diff
@@ -29,6 +32,37 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
   
+  // Store value of pointer register that points to temp. register
+  uint8_t writeBuffer = 0x00;
+
+  // Set pointer register to point to temp. register
+  i2cSendTo(devAddr, &writeBuffer, 1); 
+
+  // Store temp. value retrieved from sensor
+  uint8_t readBuffer[2];
+
+  // Read data from temp. register
+  i2cReceiveFrom(devAddr, &readBuffer[0], 2);
+
+  // Convert temp. data to Celsius
+  uint16_t tempC;
+  float multiplier = 0.125;
+
+  //1) Make into abs. value
+  if ( readBuffer[0] > 128 ) {
```

Comment:
better for readability to explicitly use a bit mask when your trying to check any bit ie (readBuffer[0] & YOUR_BIT_MASK_MACRO_NAME)